### DISCUSSION TITLE
hosts  希望添加切换和开关功能

### DISCUSSION THREAD

**Discussion by huanmin123** [he3-app/he3-feedback#281] [Ideas]
hosts  希望添加切换和开关功能  ,因为会涉及到多个环境的使用,  而且目前还不能预览最终host效果,这个可以参考下SwitchHost软件

**Comment by zhenweiwang1990** [comment]
很好的建议，我们也正在在做这个改进了。

**Comment by lyzhang1999** [comment]
感谢你的建议，已经创建了新的 issues，你可以在这里关注进展：https://github.com/he3-app/he3-feedback/issues/283
